.github/workflows/prod-build.yml

```diff
@@ -432,10 +460,11 @@ jobs:
 
       - name: Update AI Help index with macros
         working-directory: mdn/yari
-        run: yarn ai-help-macros update-index
+        run: yarn ai-help-macros update-index "$FRED_ROOT/out/en-us/docs"
```

Comment:
Nit: (dito, if that script used `$BUILD_OUT_ROOT` we wouldn't need this workaround)